### ISSUE TITLE
Config file not being read

### ISSUE THREAD

**Issue by thedamon** [causes/scss-lint#347]
I have in `project/.scss-lint.yml`

    Indentation:
        character: tab
        width: 1

and it's not picking up the config when linting `project/styles/partials/_footer.scss`

Am I doing anything wrong? Can I run a command to force it to load that config file?

**Comment by sds** [comment]
Hey @thedamon, thanks for using `scss-lint`.

You need to nest that configuration under the `linters` key. Please see https://github.com/causes/scss-lint/blob/master/config/default.yml for an example configuration.

**Comment by thedamon** [comment]
Thanks for getting back to me. I did do that while troubleshooting but forgot to update the question.  file now looks like

		linters:
		  Indentation:
		    enabled: true
		    character: tab
		    width: 1

but the rules are not being picked up

**Comment by sds** [comment]
I'm not sure what else to offer in terms of advice. When I create a file `test.scss` with the contents:

```scss
p {
  margin: 0;
}
```
(using two spaces as indentation)

...and then create an `.scss-lint.yml` file with the contents:

```yaml
linters:
  Indentation:
    enabled: true
    character: tab
    width: 1
```

Running the latest version of `scss-lint` (0.32.0 at time of writing) I get the following error as expected:

```
test.scss:2 [W] Indentation: Line should be indented with tabs, not spaces
```

Are you sure your file contains errors? Otherwise, I'm not sure what advice to offer here.

Best of luck with figuring out the issue.

**Comment by lencioni** [comment]
@thedamon can you post your entire .scss-lint.yml file here? There might be something else weird about it.

**Comment by thedamon** [comment]
It looks like I had another yml in the directory called `myproject.scss-lint.yml` that I created accidentally and didn't realize it came back when I switched branches. Once I deleted it the proper yml started being picked up (before that neither were).

Either that or it was that I rebooted Sublime Text...

Anyway it's working now!

Apologies for being a bother.
